pinot-integration-test-base/src/test/java/org/apache/pinot/integration/tests/BaseClusterIntegrationTest.java

```diff
@@ -558,7 +563,8 @@ protected void waitForDocsLoaded(long timeoutMs, boolean raiseError) {
       @Override
       public Boolean apply(@Nullable Void aVoid) {
         try {
-          return getCurrentCountStarResult() == countStarResult;
+          long currentCountStarResult = getCurrentCountStarResult();
+          return currentCountStarResult == countStarResult;
```

Comment:
(minor) seems unrelated

Reply:
yeah, I was debugging realtime cluster integration test and had to make this change for debug point. reverted it.